codegenerator/cli/templates/dynamic/codegen/src/db/Enums.res.hbs

```diff
@@ -48,14 +48,16 @@ module EntityType = {
   @genType
   type t = 
   {{#each entities as | entity |}}
-    | @as("{{entity.name.capitalized}}") {{entity.name.capitalized}}
+    | @as("{{entity.name.original}}") {{entity.name.capitalized}}
```

Comment:
Entity modules had both "key" and "name" properties. Only difference was one used original and one capitalized. I just changed this to use original and removed, key for now.

Reply:
How so?

I must double check but as far as I know this enum was only actually used in the old entity history table.

Reply:
Since there's `@genType` it's exposed to the ts users. Not sure that it's used though